osu.Game/Rulesets/Objects/Legacy/ConvertHitObjectParser.cs

```diff
@@ -32,6 +32,8 @@ public abstract class ConvertHitObjectParser : HitObjectParser
 
         protected bool FirstObject { get; private set; } = true;
 
+        protected bool AllowsLayeredHitSounds { get; set; } = true;
```

Comment:
Usually we use infinitives for these sorts of flags (so would be `AllowLayeredHitSounds`)

Reply:
My bad, i am not extremely familiar yet with the conventions used in this repo. I will change it.